global-config.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs').promises;
```

Comment:
it will break for 8 10 node versions. Not really the best we can get, why not use sync functions?